### ISSUE TITLE
False positives

### ISSUE THREAD

**Issue by TomRegan** [amperser/proselint#341]
I ran this through the release note for the code base I work on, which is an ascii format file; I got the following false reports:

>DataSource for Java was updated from 6.2.2-296613 to 6.2.5-300333.

causes

`garner.dates When specifying a date range, write 'from X to Y'.`
Wrong because it is not a valid date format.

>Resolved an issue with the margin on the spot rate being applied the

`write_good.cliches 'on the spot' is a cliché.`
Wrong because spot rate is the subject.

>When prices are nulled in the Reuters Admin applet, or the panic button is pressed

`write_good.cliches 'panic button' is a cliché.`
Again, wrong because 'panic button' is not a figure of speech.

Finally
`butterick.symbols.curly_quotes Use curly quotes “”, not straight quotes "".`
`butterick.symbols.curly_quotes Use curly quotes “”, not straight quotes "". Found 56 times elsewhere.`
`butterick.symbols.ellipsis '...' is an approximation, use the ellipsis symbol '…'.`
Wrong because of the file format.

**Comment by dialex** [comment]
I know it's been more than 4 years since this post, but I still have false positives with the curly quotes. I even tried disabling it, but proselint ignored my config file. This is my first time using it 😞 

In this screenshot you see the error, the original file with "proper" quotes, that weird env var, and my `config` (proselint ignore) file.

<img width="1162" alt="Screenshot 2021-05-29 at 14 27 18" src="https://user-images.githubusercontent.com/2140450/120072168-4aed0800-c08a-11eb-8978-0d4f033ade26.png">

**Comment by Kurt-von-Laven** [comment]
@dialex, the config file should say `typography.symbols` since you can't disable individual rules.

**Comment by suchow** [comment]
@Nytelife26 Yes, easy: "John dabbed water on the spot where the ink had splashed on his shirt." 

But I think @TomRegan is referring to the term `spot rate` (https://www.investopedia.com/terms/s/spot_rate.asp) and must have used it in something this:

> The dealer offered to execute the transaction on the spot rate quoted that morning, ensuring immediate settlement.
